packages/paste-core/components/button/src/SecondaryButton.tsx

```diff
@@ -14,28 +14,26 @@ import {DirectButtonPropTypes} from './proptypes';
 const defaultStyles: BoxStyleProps = merge(BaseStyles.default, {
   color: 'colorText',
   backgroundColor: 'colorBackgroundBody',
-  boxShadow: 'shadowBorder',
+  boxShadow: 'shadowBorderWeak',
   _hover: {
-    color: 'colorTextLinkStronger',
-    backgroundColor: 'colorBackgroundPrimaryWeakest',
-    boxShadow: 'shadowBorderPrimaryStronger',
+    color: 'colorTextPrimary',
+    backgroundColor: 'colorBackgroundBody',
+    boxShadow: 'shadowBorderPrimary',
   },
   _focus: {
-    color: 'colorTextLinkStronger',
-    backgroundColor: 'colorBackgroundPrimaryWeakest',
-    boxShadow: 'shadowFocusShadowBorder',
+    boxShadow: 'shadowFocus',
```

Comment:
issue: missing inner border on focus. Probably needs to be `shadowFocusShadowBorder` 
<img width="144" alt="image" src="https://user-images.githubusercontent.com/368249/222012158-2488789f-f510-46b3-8dde-45f122e69fa3.png">